core/modules/saver-handler.js

```diff
@@ -172,18 +179,91 @@ SaverHandler.prototype.saveWiki = function(options) {
 				$tw.notifier.display(self.titleSavedNotification);
 				if(options.callback) {
 					options.callback();
-				}
-			}
-		};
-	// Call the highest priority saver that supports this method
-	for(var t=this.savers.length-1; t>=0; t--) {
-		var saver = this.savers[t];
-		if(saver.info.capabilities.indexOf(method) !== -1 && saver.save(text,method,callback,{variables: {filename: variables.filename}})) {
-			this.logger.log("Saving wiki with method",method,"through saver",saver.info.name);
-			return true;
-		}
-	}
-	return false;
+        }
+      }
+    };
+  // Process preferred if any
+  var ignorePreferred = null
+  var preferredSaver = $tw.wiki.getTiddler("$:/config/PreferredSaver")
+  if (preferredSaver !== null && preferredSaver !== undefined) {
+    var title = preferredSaver.getFieldString("text")
+    title =
+      title === undefined || title == null || title.trim() === ''
+        ? null
+        : title.trim()
+    if (title !== null) {
+      ignorePreferred = title
+      // Process preferred saver
+      if (
+        await this.save(
+          this.getSaver(title).module,
```

Comment:
Does this code cope with there being no saver with the specified title?

Reply:
yes